src/index.tsx

```diff
@@ -35,8 +36,13 @@ if (window.ethereum) {
 }
 
 function Updaters() {
+  const location = useLocation()
+  const baseUrl = `${window.location.protocol}//${window.location.host}${location.pathname}`
```

Comment:
could we use `window.location.pathname` instead of using the hook?

Reply:
you can also use window.location.origin instead of both protocol and host https://developer.mozilla.org/en-US/docs/Web/API/Location